File: KuantumKaosYonetimi/Javascript/KuantumKaosYonetimi/VeriPaketi.js
// VeriPaketi.js
const KuantumNesnesi = require('./KuantumNesnesi');

class VeriPaketi extends KuantumNesnesi {
    constructor(id, stabilite) {
        super(id, 1, stabilite); // TehlikeSeviyesi: 1
    }

    // AnalizEt() (Çok Biçimlilik/Polymorphism)
    AnalizEt() {
        // Stabilite 5 birim düşer. Setter otomatik olarak kapsüllemeyi çalıştırır.
        this.Stabilite -= 5;
        console.log(`[ID: ${this.ID}] Veri içeriği okundu. Stabilite -5. Yeni Stabilite: ${this.Stabilite.toFixed(2)}%`);
    }
}

module.exports = VeriPaketi;

File: KuantumKaosYonetimi/CSharp/KuantumKaosYonetimi/VeriPaketi.cs
namespace KuantumKaosYonetimi
{
    // VeriPaketi: KuantumNesnesi'nden türemiştir, IKritik değildir.
    public class VeriPaketi : KuantumNesnesi
    {
        public VeriPaketi(string id, double stabilite)
            : base(id, 1, stabilite) // TehlikeSeviyesi: 1 (Düşük)
        {
        }

        // AnalizEt() (Çok Biçimlilik/Polymorphism)
        public override void AnalizEt()
        {
            // Stabilite sadece 5 birim düşer.
            Stabilite -= 5;
            Console.WriteLine($"[ID: {ID}] Veri içeriği okundu. Stabilite -5. Yeni Stabilite: {Stabilite:F2}%");
        }
    }
}